makedepend.SH

```diff
@@ -235,6 +138,8 @@ if $test -s .deptmp; then
        -e 'h; s/mini\(perlmain\)/\1/p; g' \
        .deptmp >>$mf.new
 else
+    $echo "Should not get here"
+    exit 1
```

Comment:
(Interestingly the code block this protects largely goes back to perl 1.0.)

This change however probably shouldn't be in the final patch